<html>

<head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>Rintaro Saito's Information Centre</title>
<link rel="stylesheet" href="../Styles/rs_prop2_IMM1.css" type="text/css"> 
<link rel="stylesheet" href="../Styles/each_page_style1_IMM1.css" type="text/css"> 
</head>

<body>

<div id="omega"> <!-- Whole things start from here -->
<div class="empty_bar"></div>

<div id="title_container">
   <div id="title_logo" ,="" align="right">
   <img src="../Pics/Logo/UCSD_HlSci_logo2.png">
   </div>
   <div id="main_title">
   <br/>
   <font size="5">Institute of Metabolomic Medicine</font><br>
   University of California, San Diego, School of Medicine
   </div>
   <div class="clear"></div>
</div>

<div id="navigation_container">
   <ul id="navigation_bar">
   <li><a href="../HTMLo/index.html">Home</a></li>
   <li><a href="../HTMLo/research.html">Research</a></li>
   <li><a href="../HTMLo/publications.html">Publications</a></li>
   <li><a href="../HTMLo/laboratory.html">Laboratory</a></li>
   <li><a href="../HTMLo/resources.html">Resources</a></li>
   <li><a href="../HTMLo/contact.html">Contact</a></li>
   </ul>
   <div class="clear"></div>
</div>

<div id="main_container">
   <div id="sidebar">
      <div align="center">
         <img src="../Pics/Frame/pin_gray1_1.gif"><br>
         &nbsp;
      </div>
      <div class="sidebar_categ">Our organizations</div>
      <ul class="each_list">
         <li><a href="http://sharmalab.ucsd.edu">Center for Renal Translational Medicine</a></li>
         <li><a href="http://biostats.mcc.ucsd.edu">Moores Cancer Center</a></li>
         <li><a href="http://www.ucsdbglab.org">Biochemical Genetics and Metabolomics Laboratory</a></li>
         <li><a href="http://sharpentrials.com">ClinMet</a></li>		 
      </ul>
      <div class="sidebar_categ">Kidney and nephrology</div>
      <ul class="each_list">  
         <li><a href="http://www.nephromine.org">Nephromine</a></li>
         <li><a href="http://www.kupkb.org/">KUPKB</a> The Kidney & Urinary Pathway Knowledge Base</li>
         <li><a href="http://www.dknet.org/">NIDDK Information Network</a></li>
      </ul>

	  <div class="sidebar_categ">Metabolic pathway analysis</div>
      <ul class="each_list"> 
         <li><a href="http://www.metaboanalyst.ca/">MetaboAnalyst</a> a comprehensive tool suite for metabolomic data analysis</li>
         <li><a href="http://www.msea.ca/">MSEA</a> Metabolite Set Enrichment Analysis</li>
         <li><a href="http://ekhidna.biocenter.helsinki.fi/poxo/mpea/">MPEA</a> Metabolite Pathway Enrichment Analysis</li>
         <li><a href="http://www.genome.jp/kegg/">KEGG</a> Kyoto Encyclopedia of Genes and Genomes</li>
         <li><a href="http://www.hmdb.ca/">REACTOME</a> A Curated Pathway Database</li>
         <li><a href="http://www.brenda-enzymes.info/">BRENDA</a> The Comprehensive Enzyme Information System</li>
         <li><a href="http://www.hmdb.ca/">HMDB</a> The Human Metabolome Database</li>
         <li><a href="https://xcmsonline.scripps.edu/">XCMS online</a> Scripps Ceter for Metabolomics</li>
         <li><a href="https://metlin.scripps.edu/">METLINE</a> Metabolite and Tandem MS Database, Scripps Ceter for Metabolomics</li>
         <li><a href="http://bigg.ucsd.edu/">BiGG</a> Biochemical Genetic and Genomic knowledgebase of large scale metabolic reconstructions</li>
         <li><a href="http://www.bindingdb.org/">Binding DB</a> The Binding Database</li>
      </ul>
       
      <div class="sidebar_categ">Other Software</div>
      <ul class="each_list">  
         <li><a href="http://www.cytoscape.org/">Cytoscape</a></li>
         <li><a href="http://www.r-project.org/">R</a>,
             <a href="http://rpy.sourceforge.net/rpy2.html">Rpy2</a>,
             <a href="http://rstudio.org/">RStudio</a>,
             <a href="http://www.bioconductor.org/">Bioconductor</a></li>
         <li><a href="http://www.python.org/">Python</a>,
             <a href="http://www.numpy.org/">NumPy</a>,
             <a href="http://www.scipy.org/">SciPy</a>,
             <a href="http://networkx.lanl.gov/">NetworkX</a>,
             <a href="http://pandas.pydata.org/">pandas</a>,
             <a href="http://matplotlib.org/">matplotlib</a>,
             <a href="http://www.enthought.com/">Enthought Python Distribution</a>,
             <a href="https://store.continuum.io/">Anaconda</a>
         </li><li><a href="http://www.mathworks.com/">MATLAB</a></li>
      </ul>

   </div>
   <div id="main_area">

<!-- main_area starts here -->


<H1>Metabolomics for health</H1>
<p>
<img src="../Pics/Event/ISNForefront2016_1.jpg" align="right">
Our body contains thousands of different metabolites which are involved in various molecular reactions including energy consumption, molecule synthesis and cellular signalling. Our health is maintained by the proper reactions among these metabolites. However, when we have a disease, these reactions may not occur in appropriate ways. Or conversely, if these reactions do not occur in the appropriate ways, we may have symptoms of diseases. These inappropriate reactions may be reflected by the changes in levels of some of these metabolites. We aim to characterize these changes in the patients, investigate the molecular mechanisms behind these changes, predict diseases based on these changes and find molecular drug target which may be effective in treating the diseases.
<br><br>
&#9875; <B>We are organizing 2016 ISN Forefronts Symposium "The Metabolome and Microbiome in Kidney Disease - New technologies and insights for human diseases" as a part of the International Society of Nephrology (ISN), taking place in San Diego (USA) 23-25 September, 2016.</B>
</p>



<H1>Our methodology</H1>
<p>
<img src="../Pics/Facility/GCMSPC1_2.png" align="left">
	
Our team is multidisciplinary - We have physicians, biochemists,
molecular biologists, statisticians and computational biologists as
our members. We employ various methods to investigate metabolites and
their physiological implications. Our laboratory has GC-MS
to measure ~100 metabolites in urine or blood of the patients
in high-throughput manner.
We employ various state-of-art statistical model to characterize their profiles,
and to predict outcome of patients. We also use bioinformatics to computationally
predict important enzymes and proteins. We validate the predictions using mouse models.</p>

<H1>Exciting ongoing projects</H1>
<p>Diabetic kidney disease is the leading cause of ESRD, but few biomarkers of diabetic kidney disease are available. We used gas chromatography-mass spectrometry to quantify 94 urine metabolites. We found that 13 metabolites were significantly reduced in the patients with diabetic kidney disease. Notably, organic anion transporter-1 (OAT1) knockout mice expressed a similar pattern of reduced levels of urinary organic acids, and human kidney tissue from patients with diabetic nephropathy demonstrated lower gene expression of OAT1 and OAT3. Analysis of bioinformatics data indicated that 12 of the 13 differentially expressed metabolites are linked to mitochondrial metabolism and suggested global suppression of mitochondrial activity in diabetic kidney disease. We suggest that urine metabolomics is a reliable source for biomarkers of diabetic complications, and our data suggest that renal organic ion transport and mitochondrial function are dysregulated in diabetic kidney disease.</p>


<!-- End of main_area -->

   </div>
   <div class="clear"></div>

</div>

<div id="footarea">
Engineered by Saito R 2011; Design by Saito A 2011; Contents by Saito R 2011
</div>
<div class="empty_bar"></div>
</div> <!-- Whole things ends here -->


</body>
</html>